irohad/ametsuchi/impl/rocksdb_indexer.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AMETSUCHI_ROCKSDB_INDEXER_HPP
+#define AMETSUCHI_ROCKSDB_INDEXER_HPP
+
+#include "ametsuchi/indexer.hpp"
+
+#include "ametsuchi/impl/rocksdb_common.hpp"
```

Comment:
Is not required in the header, you can use forward declarations instead

Reply:
There, the forward declaration is already used ... Header is inserted by mistake